src/app/model/ChessEngine.ts

```diff
@@ -61,7 +80,7 @@ export class ChessEngine implements ChessModel {
     );
   }
 
-  private changeTurn(side: Side): Side {
+  private changeSide(side: Side): Side {
```

Comment:
To nie zmienia strony. Zmiana strony bylaby całość:
`this.currentSide = this.currentSide === Side.WHITE ? Side.BLACK : Side.WHITE`

Wiec moze to nazwac `anotherSide`?

Reply:
jasne, zmieniam